pkg/operator/ceph/object/bucket/controller_test.go

```diff
@@ -175,7 +172,7 @@ func TestCephBucketController(t *testing.T) {
 
 		res, err := r.Reconcile(ctx, req)
 		assert.NoError(t, err)
-		assert.False(t, res.Requeue)
+		assert.True(t, res.Requeue)
```

Comment:
updated it to `True`, since I added method `opcontroller.IsReadyToReconcile()` to get cephCluster which return `requeResponse`as `WaitForRequeueIfCephClusterNotReady` when ceph health is not `ok or warn` which has `requeue` `True`

Reply:
reverted, change the code accordingly.